src/tests/ftest/util/ior_test_base.py

```diff
@@ -127,7 +127,7 @@ def _start_dfuse(self):
 
     def run_ior_with_pool(self, intercept=None, test_file_suffix="",
                           test_file="daos:testFile", update=True,
-                          create_cont=True):
```

Comment:
Rename update=True to create_pool=True, so you can do pool creation in conjunction with cont_create. As the signature of run_ior_with_pool grows, simple "update" wouldn't be clear for the user calling the method.

Reply:
Done